config/default.js

```diff
@@ -51,7 +51,8 @@ module.exports = {
   OPTIMIST_HOST: process.env.OPTIMIST_HOST || 'optimist',
   OPTIMIST_PORT: process.env.OPTIMIST_PORT || 80,
   PROPOSER_HOST: process.env.PROPOSER_HOST || 'proposer',
-  PROPOSER_PORT: process.env.PROPOSER_PORT || 8080,
+  PROPOSER_PORT: process.env.PROPOSER_PORT || 8090,
```

Comment:
I had some conflicts since client1 is running on the same port (check docker-compose). Now, why didn't I get this conflict before? I think only the Pope may know why. 

Anyway, changing ports fixes this